flink-connector-aws/flink-connector-dynamodb/src/test/java/org/apache/flink/connector/dynamodb/source/split/StartingPositionTest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.connector.dynamodb.source.split;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class StartingPositionTest {
+
+    @Test
+    void testEquals() {
+        StartingPosition startingPosition1 = StartingPosition.fromStart();
+        StartingPosition startingPosition2 = StartingPosition.fromStart();
+        StartingPosition startingPosition3 = StartingPosition.latest();
+        assertTrue(startingPosition1.equals(startingPosition2));
+        assertFalse(startingPosition1.equals(startingPosition3));
+    }
```

Comment:
Why not use the equalsVerifier?

Reply:
equalsverifier wasn't building up. So used it this way. Let me try again